src/tribler/core/utilities/tracker_utils.py

```diff
@@ -90,6 +93,8 @@ def get_uniformed_tracker_url(tracker_url):
 
             if url.scheme == 'http' and uniformed_port == HTTP_PORT:
                 uniformed_url = f'{uniformed_scheme}://{uniformed_hostname}{uniformed_path}'
+            elif url.scheme == 'https' and uniformed_port == HTTPS_PORT:
+                uniformed_url = f'{uniformed_scheme}://{uniformed_hostname}{uniformed_path}'
```

Comment:
The line `uniformed_url = f'{uniformed_scheme}://{uniformed_hostname}{uniformed_path}'` is duplicated here. It could lead to an error in the future when a developer will change one line and forget to change another.

Consider to change it to something like:

```suggestion
            if (url.scheme == 'http' and uniformed_port == HTTP_PORT) or \
                    (url.scheme == 'https' and uniformed_port == HTTPS_PORT):
                uniformed_url = f'{uniformed_scheme}://{uniformed_hostname}{uniformed_path}'
```

Reply:
Refactored it differently now.